calendar-card.js

```diff
@@ -170,12 +170,12 @@ class CalendarCard extends HTMLElement {
     }
     //${event.summary.indexOf('birthday') > 0 ? `<div class="congrats"><i class="fas fa-gift"></i>&nbsp;Send congrats</div>` : ''}
     return `
-      <div class="event-wrapper">
-        <div class="event">
+      <div class="event-wrapper" style="border:2px solid ${event.color}">
+        <div class="event" >
           <div class="info">
             <div class="summary">${event.title}</div>

```

Comment:
What about instead of doing a border, we do something like this:

```
<div class="summary">${event.title}<span style="color: ${event.color};">&nbsp&#9679;</span></div>
```

Reply:
Gives a small indication of the calendar it is on and is more in line with what you see on proper Google Calendar

Reply:
And to @rdehuyss wanting this to be configurable, we can use this:

```
<div class="summary">${event.title}
${this.config.showColors ? `<span style="color: ${event.color};">&nbsp&#9679;</span>` : ''}
</div>
```